Source/Urho3D/Core/Context.cpp

```diff
@@ -159,10 +159,11 @@ Context::Context()
 
 Context::~Context()
 {
+#ifndef MINI_URHO
+    // Save any configuration that may be unsaved at runtime.
     auto* configManager = GetSubsystem<ConfigManager>();
     configManager->SaveAll();
```

Comment:
Why is this excluded from minimal builds? I can see config manager being useful for tools.

Reply:
It fails to compile. I thought that it is by design.

Reply:
Why would it fail though? `MINI_URHO` is there to provide core utilities to tools, so it avoids compiling graphics/physics/few other things, but does include xml/json. There should be nothing that prevents it to build i think. SDL may also be omitted, not sure. But if it is - this class would still be useful when config path is explicitly specified.